apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -2208,18 +2206,13 @@ describe(`Trigger event - ${eventTriggerPath} (POST)`, function () {
       });
     });
   });
-  describe.skip('Trigger Event - [IS_MULTI_PROVIDER_CONFIGURATION_ENABLED=true]', function () {
+  describe('Trigger Event - Multi-Provider Configuration', function () {
```

Comment:
There was a period of time that the feature flags were flakey because not a perfect implementation of the feature. So it was skipped to unblock the team.
I fixed the FF service implementation during the MemoryDB migration works.